engines/gob/detection/tables_adi5.h

```diff
@@ -44,7 +44,7 @@
 		ADGF_UNSUPPORTED,
 		GUIO0()
 	},
-	kFeatures640x480,
+	kFeatures640x480, // Adi 5 actually uses 800x600 resolution (kFeatures800x600) would be required, this change will be applied when the Game is supported.
```

Comment:
Since these games aren't supported yet, it would be better to actually update the correct resolution in all the entries, instead of adding a comment.

This applies to all of the entries that use this resolution

Reply:
I already tried it in an PR but was closed https://github.com/scummvm/scummvm/pull/5271/commits/db4cbe4e4afa74722ce4a4a2f71e7cb4169e1d92#diff-86cc69f93d89aa709a768d24cf3d748d2794658ede006789ae9b35726e5447d6

Reply:
applied the changes